File: sdkwork-backend-sdk-go/types/voice_speaker_generation_input.go
package types

// 语音说话人生成输入参数
type VoiceSpeakerGenerationInput struct {
	Title string `json:"title"`
	Prompt string `json:"prompt"`
	NegativePrompt string `json:"negativePrompt"`
	ReferenceAssets []AssetMediaResource `json:"referenceAssets"`
	Storyboard VideoStoryboard `json:"storyboard"`
	Parameters map[string]interface{} `json:"parameters"`
	N int `json:"n"`
	Seed int `json:"seed"`
	Format string `json:"format"`
	Text string `json:"text"`
	SpeakerName string `json:"speakerName"`
	SpeakerId string `json:"speakerId"`
	Language string `json:"language"`
	Speed float64 `json:"speed"`
	Pitch float64 `json:"pitch"`
	Volume int `json:"volume"`
	Emotion string `json:"emotion"`
	SaveSpeaker bool `json:"saveSpeaker"`
	UsingExistingSpeaker bool `json:"usingExistingSpeaker"`
	VoiceClone bool `json:"voiceClone"`
	ReferenceAssetCount int `json:"referenceAssetCount"`
}

File: sdkwork-backend-sdk-csharp/Models/VoiceSpeakerGenerationInput.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Backend.Models
{
    public class VoiceSpeakerGenerationInput
    {
        public string? Title { get; set; }
        public string? Prompt { get; set; }
        public string? NegativePrompt { get; set; }
        public List<AssetMediaResource>? ReferenceAssets { get; set; }
        public VideoStoryboard? Storyboard { get; set; }
        public Dictionary<string, object>? Parameters { get; set; }
        public int? N { get; set; }
        public int? Seed { get; set; }
        public string? Format { get; set; }
        public string? Text { get; set; }
        public string? SpeakerName { get; set; }
        public string? SpeakerId { get; set; }
        public string? Language { get; set; }
        public double? Speed { get; set; }
        public double? Pitch { get; set; }
        public int? Volume { get; set; }
        public string? Emotion { get; set; }
        public bool? SaveSpeaker { get; set; }
        public bool? UsingExistingSpeaker { get; set; }
        public bool? VoiceClone { get; set; }
        public int? ReferenceAssetCount { get; set; }
    }
}
